### PR TITLE
:arrow_up: bump open-api-framework to 0.8.0

### PR DESCRIPTION
**Pull request by annashamray**
After the PR is merged it would be added to patch release

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -30,7 +30,8 @@ services:
       # setup_configuration env vars
       - OBJECTS_DOMAIN=web:8000
       - OBJECTS_ORGANIZATION=Objects
-      - OBJECTTYPES_API_ROOT=https://objecttypes.example.com/api/v2/
+      - OBJECTS_OBJECTTYPES_CONFIG_ENABLE=false
+      # - OBJECTTYPES_API_ROOT=https://objecttypes.example.com/api/v2/
```
Comment (stevenbal) [inline docker-compose.yml:34]: Is it intentional to disable this?

**Review by stevenbal — COMMENTED** [review]
@annashamray will you also add the warning about HSTS_SECONDS to the changelog? https://github.com/maykinmedia/open-api-framework/pull/63#discussion_r1728658426

File: docker-compose.yml (hunk 1 above)
Comment (annashamray) [reply docker-compose.yml:34]: Yes it's a temporary fix for quick-start action
`OBJECTTYPES_API_ROOT` setting was incorrect (it's fixed in this PR), but quick-start doesn't use the current build, but only the published one. 
So to make it work for now `OBJECTTYPES_API_ROOT` should be empty

**Comment by annashamray** [comment]
@stevenbal I planed to add the warning when preparing the release. Do you want me to add it now?

**Comment by stevenbal** [comment]
@annashamray no that's fine, was just to double check so we don't forget about it :+1:

**Review by stevenbal — APPROVED** [review]

**Pull request merged**